### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [ChainSafe Files/Chainsafe Files user interface](https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: packages/files-ui/src/locales/fr/messages.po [hunk 1]
```diff
@@ -408,13 +412,13 @@ msgid "Files"
 msgstr "Fichiers"
 
 msgid "Files Free"
-msgstr ""
+msgstr "Fichiers gratuits"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgstr "Files gratuits"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 2]
```diff
@@ -408,13 +412,13 @@ msgid "Files"
...
 
 msgid "Files Max"
-msgstr ""
+msgstr "Fichiers Max"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgstr "Files Max"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 3]
```diff
@@ -408,13 +412,13 @@ msgid "Files"
...
 
 msgid "Files Pro"
-msgstr ""
+msgstr "Fichiers Pro"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgstr "Files Pro"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 4]
```diff
@@ -624,7 +628,7 @@ msgid "Only send the exact amount of {0} to this address"
 msgstr "N'envoyez que la somme exacte de {0} à cette adresse"
 
 msgid "Only with annual billing"
-msgstr ""
+msgstr "Uniquement en cas de facturation annuelle"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgstr "Uniquement avec paiement annuel"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 5]
```diff
@@ -1150,6 +1154,8 @@ msgstr "Vous avez un paiement en retard. Tant qu'il n'a pas été réglé, l'acc
 
 msgid "Your account is restricted. Until you&apos;ve settled up, you can&apos;t upload any new content."
 msgstr ""
+"Votre compte est restreint. Jusqu'à ce que vous vous soyez installé(e), vous "
+"ne pouvez pas téléverser de nouveau contenu."
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"Votre compte est restreint. Merci de consulter les paramètres de paiement."
"Vous ne pouvez pas téléverser de nouveau contenu."
```

**Review by Tbaut — COMMENTED** [review]
Not particularly happy with this translation :0

**Review by Tbaut — APPROVED** [review]
Better now

File: packages/files-ui/src/locales/fr/messages.po [hunk 6]
```diff
@@ -408,13 +412,13 @@ msgid "Files"
 msgstr "Fichiers"
 
 msgid "Files Free"
-msgstr ""
+msgstr "Files gratuits"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po:415]:
```suggestion
msgstr "Files gratuit"
```

**Review by FSM1 — APPROVED** [review]

**Pull request merged**